Update repository instructions (#2155)
matrix-org/matrix.org: CONTENT.md

matrix_room = "#your-matrix-room:example.com"
licence = "PICK ONE"
latest_release = "2022-11-02"
featured = false

[extra.features]
e2ee = false
spaces = false
voip_1to1 = false
voip_jitsi = false
threads = false
sso = false
multi_account = false
multi_language = false

[extra.packages]
windows_installer = "https://example.com/download"
macos_installer = "https://example.com/download"
google_play_store.app_id = "com.example.app"
f_droid.app_id = "com.example.app"
apple_app_store = { app_id = "id1234567890", org = "example" }
flathub.app_id = "com.example.app"
webapp = "https://chat.example.com"
+++

Supercharge your communications with Example Client.
```

- For the `maturity`, please pick one of
	- `Stable` if your software has been running in production for a little while and if you are confident in the sustainability of the project
	- `Beta` if there are rough edges but people won't lose any data
	- `Alpha` if your software is still experimental
	- `Obsolete` if your software is no longer maintaines
- `featured` should be left to false. We are working on [processes to formalise which project should be featured or not](https://github.com/matrix-org/matrix.org/issues/1584).

> - For the `licence`, please use [one of the identifiers listed by the SPDX](https://spdx.org/licenses/)

Reviewer: What about licenses that are not on that list? Every once in a while I encounter those. Draupnir had that too for quite a while. Should just the name be used?

Author: Hm that's a very good question. How often did you encounter those?

I mostly want to enforce the SPDX identifier rules to have clean data and keep the door open to 🌈 hypothetical future automation 🌈  but that might be a case of premature optimisation.

I _guess_ we could allow the license name in full text if it doesn't have a SPDX identifier?

Reviewer: I encountered so far only the draupnir one but I was wondering about the national stuff like eu or german ones. Though I have to check if they maybe have ids on that list. Will check later or tomorrow

Reviewer: I think for now this is not a concern. We can revisit it when we get there

To add a hosting provider entry, add the following template to the `providers.toml`. Logos should be placed next to the the toml file. A logo should be in the SVG format. However if this is not available a PNG is acceptable provided it has sufficient resolution while also having reasonable filesize.

```toml
[[providers]]
name = "The provider's name"
image = "logo.svg"
website = "https://example.com"
operating_since = "2018"
description = """
What the service provides and any noteworthy limitations or features.
It should NOT contain blatant marketing text. We hold the right to ask for rewording it or reject it.
"""
``` 

## Adding documentation

Matrix.org provides a place to have documentation. Documentation is required to be on topic of the Matrix Protocol itself or how to get started using the protocol.

We currently split the Documentation into these types:

- Onboarding Documentation - Documentation for first time users

> - Community Documentation - Documentation which we consider is helpful to run a good community using Matrix tools. This can include guidance on things like moderation or how Matrix features benefit Communities.

Author: ```suggestion
- Community Documentation - Documentation we consider helpful to run a healthy Matrix community. This includes guidance on moderation best practices, and how Matrix features benefit communities.
```

- Technical Documentation - Documentation that help people understanding to Matrix Protocol. This can range from a rough outline on how Matrix functions to guides on things like state resolution.

> Additionally to creating a documentation there comes a responsibility to the author. We require authors of a documentation page to be reachable or delegate a new contact point if they cant maintain it any further. We will tr at least once per year to reach out to an author to verify a documentation is still accurate. This may happen more often when we notice ourself that the documentation is likely stale. If there hasn't been a reply in a reasonable timeframe we hold the right to archive a documentation page without further warning as we cannot verify all documentation on a technical level. Therefor we rather list up to date maintained documentation rather than possibly stale documentation. These archived documentations can then be found at the [Older Documentation Section](https://matrix.org/docs/older/#all-past-documentation).

Author: ```suggestion
Creating a documentation comes with the responsibility to maintain it. We require authors of a documentation page to be reachable or delegate a new contact point if they cant maintain it any further. Obviously outdated documentation with unreachable authors will be listed in the [Older Documentation Section](https://matrix.org/docs/older/#all-past-documentation).
```

### What we expect from Documentation contributions

> Documentation on matrix.org should be documentation which is part of the 3 categories we defined above. If you think that there is a category missing then please first open an issue for us to discuss this.

Author: ```suggestion
To keep documentation on matrix.org tidy and approachable, it needs to be sorted into one of the 3 categories defined above. If you think a new category is needed, please first open an issue so we can discuss it.
```

> The Documentation is expected to have a writing style. This means specifically that we expect the documentation to be written in a neutral and personal tone. This means we prefer "To do X you want to do Y" over "To get to X one should do Y". Additionally to the tone we also expect a documentation page to be structured. think about headers to use which guide the reader and make it easy to find a section in the document.

Author: ```suggestion
```

I would refrain from adding a paragraph about it until we have proper guidelines on the voice and tone :)

> Last but not least we would like to have graphics in our documentation when it makes sense for the documentation. Specifically for Onboarding and Community Documentation this helps readers to imagine things more easily. For technical documentation however this is not always possible and is ok. However complicated flows should contain visualizations of this to help readers understand it.

Author: ```suggestion
Images make concepts easier to understand than walls of text. Screenshots make the documentation easier to follow in the onboarding and community sections. Diagrams and other illustrations can help break down more technical or complicated documentation.
```

In the frontmatter, make sure to format the date as `year-month-day` (e.g. `2024-01-22`) then append `T` to specify the time as `hour:minute:seconds` (e.g. `12:30:00`) and add a final `Z` to specify that the time is on the UTC time zone. We use UTC to make it easier for humans to figure out which blog post is going to be published first if two blog posts are issued on the same day.

You can use the following template to create a new blog post:

```
+++
date = "2024-01-22T12:30:00Z"
title = "Exciting news from the Foundation!"

[taxonomies]
author = ["Jane Smith", "John Doe"]
category = ["Foundation"]
+++

## Matrix is taking over the world

We're excited to announce that our plan to conquer the world worked. We are everywhere, and people seem to be happy about it!

## Our positive impact

We are privacy-centric and don't want to track people individually, but we want to make sure we still have a positive impact. In our apps, we allow people to express their frustration whenever something doesn't work.
```

> ### Adding a picture for the socials

Reviewer: We probably should also mention the Youtube Thingy and where to put images related to the posts itself. (as in not Banner but content)

Author: Oh very good
